mentors/elixir/m.json

```
@@ -1 +1,10 @@
-[]
+[
+    {
+    "github_username": "mnhdrn",
+    "name": "MonoHedrion",
+    "link_text": null,
+    "link_url": "https://mnhdrnwebsite.herokuapp.com/",
+    "avatar_url": "https://avatars3.githubusercontent.com/u/25202393?s=400&u=2bf6fe08f03c8cf0ad15debf5fbd1874d7f748db&v=4",
+    "bio": "I'm currently a DevOps & Instructor for Red Hat, but I love functional programming !"
+  },
```

```suggestion
  }
```
Trailing comma not needed here.